deformable_manipulation_experiment_params/launch/cloth_single_pole_params.launch

```diff
@@ -29,21 +32,22 @@
     <param name="table_surface_z"       value="$(arg table_surface_z)"  type="double"/>
 
     <!-- Cloth size params -->
-    <arg name="cloth_x_size"                    value="0.3"/>
-    <arg name="cloth_y_size"                    value="0.5"/>
-    <arg name="cloth_num_control_points_x"      value="30"/>
-    <arg name="cloth_num_control_points_y"      value="50"/>
+    <arg name="cloth_x_size"                    default="0.3"/>
```

Comment:
We need the values to be what they are in master here for the experiment to function as seen in the paper. If you want some other values, I suggest you create a new launch file, or just change these only in your branch, not in master.